sound/soc/sof/intel/byt.c

```diff
@@ -508,7 +508,7 @@ static struct snd_soc_dai_driver byt_dai[] = {
  * Probe and remove.
  */
 
-#if IS_ENABLED(CONFIG_SND_SOC_SOF_EDISON)
+#if IS_ENABLED(CONFIG_SND_SOC_SOF_MERRIFIELD)
 
 static int byt_pci_probe(struct snd_sof_dev *sdev)
```

Comment:
Same question here

Reply:
we should probably rename it to merrifield_pci_probe to avoid the confusion.

Reply:
@plbossart, `mrfld` since it's used as a abbreviation AFAICS.